src/.vuepress/theme/components/PrimaryLink.vue

```diff
@@ -0,0 +1,18 @@
+<template>
```

Comment:
Good idea! i will use it in this [PR](https://github.com/stratum-mining/stratumprotocol.org/pull/29) and improve it to handle internal links